itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcidTables.java

```diff
@@ -809,7 +810,7 @@ public void testOpenTxnEvent() throws Throwable {
             primary.dump(primaryDbName);
 
     long lastReplId = Long.parseLong(bootStrapDump.lastReplicationId);
-    primary.testEventCounts(primaryDbName, lastReplId, null, null, 22);
+    primary.testEventCounts(primaryDbName, lastReplId, null, null, 16);
```

Comment:
How many dump open/commit considered here?

Reply:
Two dump and one load operation is not logged that totals to 6 events not logged (OpenTxn and CommitTxn)

Reply:
primary.testEventCounts would count the number of events captured after the bootstrap was initiated. In this patch, we are not capturing openTxn and commitTxn for hive repl Txns. So, the event count is reduced to 16.